dashboard/functions.py

```diff
@@ -232,20 +232,22 @@ def create_genre_bar_chart(show_nsfw: bool, os_selection: list[str], start_date:
     return genre_bar_graph
 
 
-
 def preprocess_descriptions(descriptions: list[str]) -> dict[str, int]:
     """
-    Preprocess game descriptions: lowercasing, removing punctuation, filtering stopwords.
+    Preprocess game descriptions: lowercasing, removing punctuation, filtering stopwords,
+    and excluding specific words.
     """
     stop_words = set(nltk_stopwords.words('english'))
+    custom_excluded_words = {"game", "play","new", "must", "get", "one", 
```

Comment:
This is an interesting list of extra stopwords. We can always add to it later if there appear to be more redundant words in the cloud.